app/templates/client/index.html

```diff
@@ -50,7 +50,11 @@
       <script src="socket.io-client/socket.io.js"></script><% } %>
     <!-- endbuild -->
 
+        <% if(filters.babel) { %>
+        <!-- build:js(.tmp) app/app.js -->
```

Comment:
What does this do exactly? Why is it different depending on wether babel is used?